### PR TITLE
Create new experiment version async

### LINKED ISSUE #901: Create versions async

### PR DESCRIPTION
**Pull request by SmittieC**
Resolves #901 

## Description

Since creating versions of router experiments with assistant children and many files bombed out (gateway timeout), we're now creating versions async.

## User Impact

users should be able to create versions of large assistants. Should being the keyword here

### Demo

Will include one once the UI is done

### CODE REVIEW

**Comment by SmittieC** [comment]
Ready

**Comment by SmittieC** [comment]
I'm not convinced that we should be creating a bunch of tasks in the management command, but it will fail for some assistants if we do not. Maybe we should be smarter in the management command and figure out when to give it to a task? I propose:
1. We see how this does with the large assistants
2. If it goes well, we can create versions async for experiments with assistants that have > X files.

**Review by stephherbers — APPROVED** [review]

File: apps/experiments/tasks.py [hunk 1]
```diff
@@ -15,6 +15,13 @@
 from apps.utils.taskbadger import update_taskbadger_data
 
 
+@shared_task(bind=True, base=TaskbadgerTask)
+def async_create_experiment_version(self, experiment_id: int, *arg, **kwargs) -> dict:
```
Comment (snopoke) [inline apps/experiments/tasks.py:19]: nit: specify the kwargs explicitly

**Review by snopoke — CHANGES_REQUESTED** [review]
Currently this won't give any feedback that something is happening behind the scenes. I think it's likely users will do the workflow repeatedly until they see something on the UI change.

My thought is that once the user clicks the button, the form is submitted using HTMX and the page content is replaced with a progress bar which polls for task completion.

File: apps/experiments/tasks.py (hunk 1 above)
Comment (SmittieC) [reply apps/experiments/tasks.py]: Addressed both of your comments (see the demo)

File: apps/experiments/models.py [hunk 2]
```diff
@@ -588,6 +588,7 @@ class Experiment(BaseTeamModel, VersionsMixin):
     )
     debug_mode_enabled = models.BooleanField(default=False)
     citations_enabled = models.BooleanField(default=True)
+    create_version_task_id = models.CharField(max_length=128, blank=True)
```
Comment (SmittieC) [inline apps/experiments/models.py:591]: This field's value is not really being used directly, but I think we should keep it like this. Might come in handy if we're wanting to debug something

**Review by snopoke — APPROVED** [review]

**Pull request merged**